### ISSUE TITLE
tsconfig doesn't seem to be working

### ISSUE THREAD

**Issue by appsforartists** [bazelbuild/rules_typescript#51]
As instructed, I have this in my root `BUILD`:

```python
exports_files(["tsconfig.json"])
```

and this in my package BUILD:

```python

ts_library(
  name = "core",
  module_name = "material-motion",
  module_root = "src",
  srcs = glob(
    include = ["src/**/*.ts"],
    exclude = ["**/__tests__/*"],
  ),
  tsconfig = "//:tsconfig.json",
)
```

However, the options in my tsconfig aren't being respected.  No matter what I do, I'm getting es5 output.  Moreover, I can replace `//:tsconfig.json` with any garbage name, but it is silently ignored.

Here's `//:tsconfig.json`:

```json
{
  "compilerOptions": {
    "declaration": true,
    "lib": [
      "es2015",
      "es2016",
      "es2017",
      "dom"
    ],
    "importHelpers": true,
    "module": "commonjs",
    "moduleResolution": "node",
    "noImplicitAny": true,
    "noImplicitThis": true,
    "sourceMap": true,
    "strictNullChecks": true,
    "target": "es2015"
  },
  "bazelOptions": {
    "googmodule": true
  }
}
```

**Comment by alexeagle** [comment]
You cant control --target. It should warn you about that, see https://github.com/bazelbuild/rules_typescript/issues/34

Getting other output is a concern of another rule, see the es6_output example.

Are you saying you can reference an invalid label in the tsconfig attribute and Bazel doesn't error?

**Comment by appsforartists** [comment]
Yup

**Comment by alexeagle** [comment]
okay well re-open if you figure out how to repro :)
